<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Booking</class>
 <widget class="QDialog" name="Booking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius:20;
</string>
       </property>
       <property name="title">
        <string>BOOKING</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QPushButton" name="btnBook">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#btnBook
{
background-color: rgb(79, 143, 0);
border-radius:10px;
}
#btnBook:hover
{
background-color: rgb(115, 252, 214);
}</string>
        </property>
        <property name="text">
         <string>BOOK</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblInfo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Select Slot :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbSlot">
           <property name="styleSheet">
            <string notr="true">#cmbslot
{
background-color: rgb(200, 136, 33);

}
</string>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>120</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Service:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbslot_facilities">
           <property name="styleSheet">
            <string notr="true">#cmbslot
{
background-color: rgb(200, 136, 33);

}
</string>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>AVAILABLE SLOTS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>260</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Available :  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">#label_7{background:white}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>300</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Unavailable :  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">#label_9{background:black}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>451</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="lbl1">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>750</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>10:30 AM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl2">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl3">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>11:30 AM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl4">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl5">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>12.30 PM</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="lbl6">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>01:00 PM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl7">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>01:30 PM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl8">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>02:00 PM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl9">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>02:30 PM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl10">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>03.00 PM</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="lbl11">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>03:30 PM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl12">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>04:00 PM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl13">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>04:30 PM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl14">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>05:00 PM</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
